src/routers/OrganizationRouter.ts

```diff
@@ -122,6 +124,212 @@ const getAdminRouter = (basePath: string = '/', routePath: string = '/management
     })
   );
 
+  router.post(
+    path,
+    guard.enforcePrimaryGroup(true),
+    AuthzGuards.writeOrganizationsGuard,
+    asyncHandler(async (req: AuthzRequest, res: Response) => {
+      const authzService: AuthzService = req.app.locals.authzService;
+      const authMgmtService: AuthManagementService = req.app.locals.authManagementService;
+
+      const { name, description, owners } = req.body;
+
+      const groupName = name.trim().toUpperCase();
+      const groupDescription = description.trim();
+
+      if (!groupName) {
+        throw new ParameterRequiredError('Invalid name.', 400);
+      }
+
+      if (!groupDescription) {
+        throw new ParameterRequiredError('Invalid description.', 400);
+      }
+
+      if (!Array.isArray(owners)) {
+        throw new ParameterRequiredError('Invalid owners.', 400);
+      }
+
+      const ownerEmail = owners[0];
+
+      const user = await authMgmtService.getUserByEmail(ownerEmail);
+
+      if (!user) {
+        throw new UserNotFoundError('Invalid owner specified.', 404);
+      }
+
+      const SCOPES_READ = [
```

Comment:
Might pull this in a separate service, where we can define interfaces for create/delete organizations.

Doesn't need to be handled in this PR.

Reply:
ok.